models/model.js

```diff
@@ -63,4 +63,41 @@ exports.updateVotebyArticleId = (voteChange, article_id, voteObjKey) => {
             const updatedArticle = result.rows[0];
             return updatedArticle;
         })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = (topic) => {
+
+    const sort_by = 'created_at';
+    const order = 'DESC';
+
+    if (!['cats', 'mitch', 'paper'].includes(topic) && topic !== undefined) {
+        return Promise.reject({ status: 400, msg: "Topic doesn\'t exist!" });
```

Comment:
these are the topics included in the test data... however the dev data we use in production may contain more topics, and there may come a time when you want to add more topics, can you think of a way to use SQL to check the database for a topic that exists / doesn't exist ?